stories/timeline/props-table.tsx

```diff
@@ -102,16 +102,52 @@ const DATA: ApiTableSectionProps[] = [
     {
         name: "TimelineStatusProps",
         attributes: [
+            {
+                name: "",
+                description: (
+                    <>
+                        <b>DEPRECATED</b>. This component will be replaced
+                        by&nbsp;
+                        <a
+                            href="https://designsystem.life.gov.sg/react/index.html?path=/docs/modules-pill--docs"
+                            rel="noreferrer"
+                            target="_blank"
+                        >
+                            PillProps
+                        </a>
+                    </>
+                ),
+            },
             {
                 name: "type",
                 mandatory: true,
-                description: "The color style of the pill",
+                description: (
+                    <>
+                        The color style of the pill
+                        <br />
+                        <b>DEPRECATED</b>. Use&nbsp;
+                        <a
```

Comment:
Instead of linking, should just list them out. The deprecation notice is confusing here also. Should perhaps say it as such:

The color style of the pill. **Note**: from v2.4.0 onwards, values `"dark"` and `"light"` will no longer be supported in favour of `"solid"` and `"outline"` respectively

Reply:
Thanks for the the clear explaination, now I have better understanding on what should be done in this change and future change.